app/lib/common/tutorial_dialogs/bottom_navigation_tutorials/bottom_navigation_tutorials.dart

```diff
@@ -0,0 +1,93 @@
+import 'package:acter/common/themes/app_theme.dart';
+import 'package:acter/common/tutorial_dialogs/show_tutorials.dart';
+import 'package:acter/common/tutorial_dialogs/space_overview_tutorials/create_or_join_space_tutorials.dart';
+import 'package:acter/common/tutorial_dialogs/target_focus.dart';
+import 'package:acter_flutter_sdk/acter_flutter_sdk.dart';
+import 'package:flutter/material.dart';
+import 'package:tutorial_coach_mark/tutorial_coach_mark.dart';
+import 'package:flutter_gen/gen_l10n/l10n.dart';
+
+final dashboardKey = GlobalKey();
+final updateKey = GlobalKey();
+final chatsKey = GlobalKey();
+final activityKey = GlobalKey();
+final jumpToKey = GlobalKey();
+
+const bottomNavigationPrefKey = 'bottomNavigationPrefKey';
+
+Future<void> onSkip() async {
+  final prefs = await sharedPrefs();
+  if (prefs.getBool(bottomNavigationPrefKey) ?? true) {
+    await prefs.setBool(bottomNavigationPrefKey, false);
+  }
+}
+
+void showCreateOrJoinSpaceTutorials(BuildContext context) {
+  if (isDesktop) createOrJoinSpaceTutorials(context: context);
+}
+
+void bottomNavigationTutorials({required BuildContext context}) async {
+  final prefs = await sharedPrefs();
+  final isShow = prefs.getBool(bottomNavigationPrefKey) ?? true;
```

Comment:
Activity, `bottomNavigationPrefKey` is set to `false` on both cases  (`onSkip` as well as `onFinish`),  check the below screenshot.

![Screenshot 2024-04-11 at 12 49 35 PM](https://github.com/acterglobal/a3/assets/51526480/f1a34333-7f36-4acb-9d3c-7f9c1f9c0384)


Confusion is due to bad naming of the generic method (`onSkip()`). Let me change the name so that it won't be confusing. :)